monitor/sth_fetcher_test.go

```diff
@@ -38,6 +38,13 @@ func TestObserveSTH(t *testing.T) {
 	// Make an STH observation
 	m.fetcher.observeSTH()
 
+	// Check STH fetches are counted
+	fetchObservations := test.CountCounterVecWithLabels(m.fetcher.stats.sthFetchTotal, labels)
```

Comment:
Sorry, I think the way Github applied my suggestion left this line too. L42 should be deleted here since its now combined into L43.